lab/login/style.css

```
@@ -12,6 +13,14 @@
   font-family: 'JetBrains Mono', sans-serif;
 }
 
+#user-lord\ cat > strong {
+  background: linear-gradient(to right, #c5a4ed, #a4a5ed 100%);
+  -webkit-text-fill-color: transparent;
+  background-clip: text;
+  font-family: 'UnifrakturMaguntia';
```

```suggestion
  font-family: 'UnifrakturMaguntia';
  font-size: 20px;
```

lets make sure its big enough to read